### DISCUSSION TITLE
OTEL logback appender ignores additivity?

### DISCUSSION THREAD

**Discussion by den850x** [open-telemetry/opentelemetry-java-instrumentation#12943] [Q&A] [answered]
My logback.xml defines a custom logger that has additivity="false", and the OtelAppender is configured for the root only.
But the messages logged to MyPrivateLogger are still getting picked up by OtelAppender.
Is this expected?

```
 <logger name="MyPrivateLogger" additivity="false" level="INFO">
        <appender-ref ref="Console"/>
 </logger>

  <root level="INFO">
    <appender-ref ref="OtelAppender"/>
  </root>
```

**Comment by laurit** [comment]
Please provide a minimal application that reproduces your issue.

**Reply by den850x** [reply, marked as answer]
Turned out, my issue was caused by running both OTEL Logback Appender and [OTEL JavaAgent](https://opentelemetry.io/docs/zero-code/java/agent/). The latter had [logback integration](https://opentelemetry.io/docs/zero-code/java/agent/disable/#suppressing-specific-agent-instrumentation) enabled by default. 

The log output from `MyPrivateLogger` was picked up by the OTEL Java Agent. 
I've disabled its logback integration by setting `otel.instrumentation.logback-appender.enabled=false`, so now I only have OTEL Logback Appender running in my app. This fixed my issue.

Sorry for the confusion!
